### PR TITLE
Preserve prep-file output paths from successful jobs

### PR DESCRIPTION
**Pull request by charles-cowart**
Unit-tests forthcoming.

### CODE REVIEW

**Comment by charles-cowart** [comment]
@antgonza ready for review!

File: sequence_processing_pipeline/GenPrepFileJob.py [hunk 1]
```diff
@@ -66,11 +67,28 @@ def __init__(self, run_dir, convert_job_path, qc_job_path, output_path,
...
-        self.command = [self.seqpro_path,
+        self.command = [self.seqpro_path, '--verbose',
                         join(self.output_path, self.run_id),
                         f'"{self.sample_sheet_path}"',
                         join(self.output_path, 'PrepFiles')]
 
+    def _get_prep_file_paths(self, stdout):
+        tmp_l = stdout.split('\n')
+        tmp_l = [x for x in tmp_l if x != '']
+        tmp_d = {}
```
Comment (antgonza) [inline sequence_processing_pipeline/GenPrepFileJob.py:78]: Can you use a defaultdict(list) here to reduce the number of lines?

File: sequence_processing_pipeline/GenPrepFileJob.py [hunk 2]
```diff
@@ -66,11 +67,28 @@ def __init__(self, run_dir, convert_job_path, qc_job_path, output_path,
...
+        print(tmp_l)
```
Comment (antgonza) [inline sequence_processing_pipeline/GenPrepFileJob.py:79]: is this print needed?

**Review by antgonza — CHANGES_REQUESTED** [review]

File: sequence_processing_pipeline/GenPrepFileJob.py (hunk 2 above)
Comment (charles-cowart) [reply sequence_processing_pipeline/GenPrepFileJob.py:79]: doh!

File: sequence_processing_pipeline/GenPrepFileJob.py (hunk 1 above)
Comment (charles-cowart) [reply sequence_processing_pipeline/GenPrepFileJob.py]: ty! I need to remember to use this more.

**Review by antgonza — APPROVED** [review]

**Pull request merged**